jsonschema-core.xml

```diff
@@ -2757,11 +2756,11 @@
                     Implementations MAY elect to provide additional information.
                 </t>
 
-                <section title="Keyword Relative Location">
+                <section title="Evaluation path">
                     <t>
-                        The relative location of the validating keyword that follows the path
-                        traversed through the schema. The value MUST be expressed as a JSON
-                        Pointer, and it MUST include any by-reference applicators such as
+                        The location of the schema resource that produced the output unit, represented
```

Comment:
I like this a lot better, although I still have some thoughts :-)

1.  Should be "schema object" and not "schema resource" - that's an important distinction.

2.  Maybe instead of "location" this should just be "The path traversed to reach the schema object that produced the output unit"?  Because it's not really a location in the same sense as the other two, _unless_ we very clearly define something like "dereferenced evaluation structure" and then say "The location within the dereferenced evaluation structure".

Reply:
1. 👍 
2. #1264 defines "evaluation path" as an actual term (see the first change, line 682).  I think this really just needs to echo that.  You're right, though, in that it's not a location, _per se_.  I'll make an update.